src/main/java/org/jitsi/videobridge/RtpChannel.java

```diff
@@ -1412,6 +1412,8 @@ public void setRtpHeaderExtensions(
     {
         if ((rtpHeaderExtensions != null) && (rtpHeaderExtensions.size() > 0))
         {
+            stream.clearRTPExtensions();
```

Comment:
interesting, so not having this before was a bug, ya?

Reply:
Not a bug, it was just not being used.